### PR TITLE
Ahd develop

### PR DESCRIPTION
**Pull request by masoudmanson**

### CODE REVIEW

**Review by Farhad7d7 — APPROVED** [review]
Done

File: app/src/main/java/ir/fanap/chattestapp/application/ui/function/FunctionFragment.kt [hunk 1]
```diff
@@ -3208,7 +3210,7 @@ class FunctionFragment : Fragment(),
         var requestDeleteMessage = RequestDeleteMessage
             .Builder()
             .messageIds(deleteListIds)
-            .deleteForAll(false)
```
Comment (Farhad7d7) [inline app/src/main/java/ir/fanap/chattestapp/application/ui/function/FunctionFragment.kt]: ?

File: app/src/main/java/ir/fanap/chattestapp/application/ui/function/FunctionFragment.kt [hunk 2]
```diff
@@ -1885,9 +1885,11 @@ class FunctionFragment : Fragment(),
 
             val position = getPositionOf(ConstantMsgType.SPAM_THREAD)
 
-            changeFunTwoState(position, Method.DONE)
+            changeFunOneState(position, Method.DONE)
 
-            changeFunThreeState(position, Method.RUNNING)
+          //  changeFunTwoState(position, Method.RUNNING)
```
Comment (Farhad7d7) [inline app/src/main/java/ir/fanap/chattestapp/application/ui/function/FunctionFragment.kt]: FunTwoState should change to RUNNING

**Review by Farhad7d7 — CHANGES_REQUESTED** [review]
Check Comments

File: app/src/main/java/ir/fanap/chattestapp/application/ui/function/FunctionFragment.kt (hunk 1 above)
Comment (ahd2254) [reply app/src/main/java/ir/fanap/chattestapp/application/ui/function/FunctionFragment.kt]: ok

File: app/src/main/java/ir/fanap/chattestapp/application/ui/function/FunctionFragment.kt (hunk 2 above)
Comment (ahd2254) [reply app/src/main/java/ir/fanap/chattestapp/application/ui/function/FunctionFragment.kt]: ok

**Pull request merged**